geomesa-convert/geomesa-convert-xml/src/test/scala/org/locationtech/geomesa/convert/xml/XMLConverterTest.scala

```diff
@@ -0,0 +1,171 @@
+package org.locationtech.geomesa.convert.xml
+
+import com.typesafe.config.ConfigFactory
+import org.junit.runner.RunWith
+import org.locationtech.geomesa.convert.SimpleFeatureConverters
+import org.locationtech.geomesa.convert.Transformers.EvaluationContext
+import org.locationtech.geomesa.utils.geotools.SimpleFeatureTypes
+import org.specs2.mutable.Specification
+import org.specs2.runner.JUnitRunner
+
+@RunWith(classOf[JUnitRunner])
+class XMLConverterTest extends Specification {
+
+  val sftConf = ConfigFactory.parseString(
+    """{ type-name = "xmlFeatureType"
+      |  attributes = [
+      |    {name = "number", type = "Integer"}
+      |    {name = "color",  type = "String"}
+      |    {name = "weight", type = "String"}
+      |    {name = "source", type = "String"}
+      |  ]
+      |}
+    """.stripMargin)
+
+  val sft = SimpleFeatureTypes.createType(sftConf)
+  implicit val ec = new EvaluationContext(null, null)
+
+  "XML Converter" should {
+
+    "parse multiple features out of a single document" >> {
+      val xml =
+        """<doc>
+          |  <DataSource>
+          |    <name>myxml</name>
+          |  </DataSource>
+          |  <Feature>
+          |    <number>123</number>
+          |    <color>red</color>
+          |    <physical weight="127.5" height="5'11"/>
+          |  </Feature>
+          |  <Feature>
+          |    <number>456</number>
+          |    <color>blue</color>
+              <physical weight="w2" height="h2"/>
+          |  </Feature>
+          |</doc>
+        """.stripMargin
+
+      val parserConf = ConfigFactory.parseString(
+        """
+          | converter = {
+          |   type         = "xml"
+          |   id-field     = "uuid()"
+          |   feature-path = "Feature" // can be any xpath - relative to the root, or absolute
+          |   fields = [
+          |     // paths can be any xpath - relative to the feature-path, or absolute
+          |     { name = "number", path = "number",           transform = "$0::integer" }
+          |     { name = "color",  path = "color",            transform = "trim($0)" }
+          |     { name = "weight", path = "physical/@weight", transform = "trim($0)" }
+          |     { name = "source", path = "/doc/DataSource/name/text()" }
```

Comment:
It doesn't - I left it in there just b/c that's how a lot of xpath examples are. I think technically it is more correct, in that it selects the text of the node vs the node itself. I was just trying to provide some different examples of xpath expressions.